extension/resources/walkthrough/dvc-learn-more.md

```diff
@@ -6,9 +6,9 @@
 
 - Browse the [user guide](https://dvc.org/doc/user-guide).
 
-- View all [available commands](https://dvc.org/doc/command-reference).
+- View all [available `dvc` commands](https://dvc.org/doc/command-reference).
 
```

Comment:
This file just re-lists the side bar og dvc.org/doc. We can just link there instead and remove this, I think.